### PR TITLE
Add Overamped by @JosephDuffy

### PR DESCRIPTION
**Pull request by bradenglund**
1. [x] Project URL:https://github.com/JosephDuffy/Overamped
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [ ] Screenshot included 
5. [x] Avoid iOS or open-source in description as it is assumed
6. [x] Use this commit title format if applicable: Add app-name by @github-username
7. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -26759,5 +26759,17 @@
       ],
       "stars": 186
     }
+     {
```
Comment (dkhamsing) [inline contents.json:26762]: @bradenglund hey, missing a comma here - can you add `,` before `{`

thank you!

Comment (bradenglund) [reply contents.json]: Added

**Comment by dkhamsing** [comment]
@bradenglund thank you!

**Comment by dkhamsing** [comment]
Screenshots

Screen Shot 2021-10-01 at 7 46 21 AM

**Pull request merged**